externs/ima.js

```diff
@@ -227,11 +227,18 @@ google.ima.AdEvent.Type = {
 
 /**
  * @typedef {{
+ *   adsResponse: ?string,
```

Comment:
I think the type should be `string|undefined`, this will allow the field to be missing by the compiler.  The other should be the same.

Reply:
Done